packages/svelte-ast-print/src/fragment.test.ts

```diff
@@ -100,17 +108,20 @@ describe(printFragment, () => {
 
 	it("it prints correctly fragment code with typescript syntax", ({ expect }) => {
 		const code = `
-			<script lang="ts">
-				//
-			</script>
+<script lang="ts">
+	//
+</script>
 
-			{#snippet template({ children, ...args }: Args<typeof Story>, context: StoryContext<typeof Story>)}
-				<Button {...args}>{children}</Button>
-			{/snippet}
+{#snippet template({ children, ...args }: Args<typeof Story>, context: StoryContext<typeof Story>)}
+	<Button {...args}>{children}</Button>
+{/snippet}
 		`;
 		const node = parse_and_extract<AST.Fragment>(code, "Fragment");
 		expect(printFragment(node).code).toMatchInlineSnapshot(`
-			"{#snippet template({ children, ...args }: Args<typeof Story>, context: StoryContext<typeof Story>)}
+			"
```

Comment:
Intentional. I'm not touching whitespace characters if we're to use `printFragment()` directly.

It's only being removed from the `printRoot()` and above.